scripts/devenv-builder/configure-vm.sh

```diff
@@ -136,8 +136,8 @@ if ${INSTALL_BUILD_DEPS} || ${BUILD_AND_RUN}; then
     if ${DNF_UPDATE}; then
         "${DNF_RETRY}" "update"
     fi
-    "${DNF_RETRY}" "install" "gcc git golang cockpit make jq selinux-policy-devel rpm-build jq bash-completion avahi-tools createrepo"
-    sudo systemctl enable --now cockpit.socket
+    # "${DNF_RETRY}" "install" "gcc git golang cockpit make jq selinux-policy-devel rpm-build jq bash-completion avahi-tools createrepo"
+    # sudo systemctl enable --now cockpit.socket
```

Comment:
@ggiguash , we need this part handled on different PR,
we dont want to enable cockpit when it is running in build-only mode.
its creating issue for containers build (systemd is not running during build time)